t/sharness.d/flux-sharness.sh

```diff
@@ -33,6 +33,29 @@ test_size_large() {
 }
 
 
+#
```

Comment:
commit message english error, "does we want" -> "does what we want"

Reply:
Thanks! Fixed and pushed